client/src/modules/ward_module/ward_module.routes.js

```diff
@@ -0,0 +1,26 @@
+angular.module('bhima.routes')
+  .config(['$stateProvider', wardModuleStateProvider]);
+
+function wardModuleStateProvider($stateProvider) {
+  $stateProvider
+    .state('wardModuleConfiguration', {
+      url : '/ward_module/configuration',
```

Comment:
I think that `/wards/configuration` is cleaner than `ward_module/configuration`.  How do you feel about switching this URL to `wards`?

Reply:
😄 there was already the previous `/ward`, i will fix that 👍